src/main/java/org/jabref/gui/exporter/ExportToClipboardAction.java

```diff
@@ -77,29 +84,21 @@ private String exportToClipboard(Exporter exporter) {
 
             // Write to file:
             exporter.export(panel.getBibDatabaseContext(), tmp,
-                    panel.getBibDatabaseContext()
-                         .getMetaData()
-                         .getEncoding()
-                         .orElse(Globals.prefs.getDefaultEncoding()),
-                    entries);
+                            panel.getBibDatabaseContext()
```

Comment:
Do we really need to export to a tmp file? The exporter should be able to work on a output stream and then we can simply pass a StringOutputBuffer here. I know this may be a bigger task, so take it as a suggestion and not requirement for merge ;-)

Reply:
It's out of scope and I think we have some other more important bugs to fix atm.